test/sql/debug_helpers.sql

```diff
@@ -1,8 +1,9 @@
-
 CREATE EXTENSION IF NOT EXISTS vector;
 CREATE EXTENSION IF NOT EXISTS lanterndb;
+
 -- initially, before we create an index, the variable does not exist
-\ir test_helpers/small_world.sql
+\ir sql/test_helpers/small_world.sql
+\ir sql/test_helpers/common.sql
```

Comment:
Where is `common.sql`? What is it for?

Can you make sure this works with `livedebug.py`?
I think currently there is an issue that we cannot `livedebug` any test that `\ir`s another file because the file path is not resolved correctly. 
We should make sure `livedebug` works for this case or think of a different way to include sql test dependencies.

Reply:
The `common.sql` was previously executed from `run_all_tests.sh` file before each test case.
Yes currently this is not going to work for `livedebug` script, I'll try to be sure that test suits works with livedebug